src/main/resources/swagger/api-docs.yaml

```diff
@@ -679,6 +679,12 @@ definitions:
       machineConfig:
         description: The machine configurations for the master and worker nodes
         $ref: '#/definitions/MachineConfig'
+      stopAfterCreation:
```

Comment:
@zarsky-broad @MattPutnam @bradtaylor fyi

I implemented this as a new optional flag in the "create cluster" JSON request. If not set, it defaults to the current behavior.

Reply:
cc @calbach @blrubenstein 

I know AoU is not stopping clusters currently, just letting you know this flag is being added. Btw we're also starting work on the stop-after-idle feature.